utils/tests/src/main/java/io/apicurio/registry/utils/tests/RegistryServiceExtension.java

```diff
@@ -89,21 +93,34 @@ public boolean supportsTestTemplate(ExtensionContext context) {
         String registryUrl = TestUtils.getRegistryUrl(rst);
 
         ExtensionContext.Store store = context.getStore(ExtensionContext.Namespace.GLOBAL);
-        RegistryServiceWrapper plain = store.getOrComputeIfAbsent(
-            "plain_client",
-            k -> new RegistryServiceWrapper(k, "create", registryUrl),
-            RegistryServiceWrapper.class
-        );
-        RegistryServiceWrapper cached = store.getOrComputeIfAbsent(
-            "cached_client",
-            k -> new RegistryServiceWrapper(k, "cached", registryUrl),
-            RegistryServiceWrapper.class
-        );
-
-        return Stream.of(
-            new RegistryServiceTestTemplateInvocationContext(plain),
-            new RegistryServiceTestTemplateInvocationContext(cached)
-        );
+
+        List<TestTemplateInvocationContext> invocationCtxts = new ArrayList<>();
+
+        if (testRegistryClient(REGISTRY_CLIENT_CREATE)) {
+            RegistryServiceWrapper plain = store.getOrComputeIfAbsent(
+                    "plain_client",
+                    k -> new RegistryServiceWrapper(k, REGISTRY_CLIENT_CREATE, registryUrl),
+                    RegistryServiceWrapper.class
+                );
+            invocationCtxts.add(new RegistryServiceTestTemplateInvocationContext(plain));
+        }
+
+        if (testRegistryClient(REGISTRY_CLIENT_CACHED)) {
+            RegistryServiceWrapper cached = store.getOrComputeIfAbsent(
+                    "cached_client",
+                    k -> new RegistryServiceWrapper(k, REGISTRY_CLIENT_CACHED, registryUrl),
+                    RegistryServiceWrapper.class
+                );
+            invocationCtxts.add(new RegistryServiceTestTemplateInvocationContext(cached));
+        }
+
+        return invocationCtxts.stream();
+    }
+
+    private boolean testRegistryClient(String clientType) {
+        String testRegistryClients = TestUtils.getTestRegistryClients();
+        return testRegistryClients == null || testRegistryClients.equalsIgnoreCase("all")
+                || testRegistryClients.equalsIgnoreCase(clientType);
```

Comment:
Can you make "all" one of the java constants along with "create" and "cached"?

Reply:
done :)